backend/src/main/java/ai/verta/modeldb/batchProcess/OwnerRoleBindingRepositoryUtils.java

```diff
@@ -124,6 +125,12 @@ private static void migrateRepositories() {
 
         transaction.commit();
         lowerBound += pagesize;
+      } catch (Exception ex) {
```

Comment:
not related directly to the PR but can you please add comments on the top of the files in batchProcess/* on what they are doing?

Reply:
Sure. it makes sense.